evasquez26/angular4-startup
# S2-ONE-122 Request Role Option

## Story / Task
A user should be able to see their roles in organizations and an option to request a role should be available
see [ONE-122](https://bixlabs.atlassian.net/browse/ONE-122)


## What this PR does?

Add ability to see a list of your roles and an option to request a new role


### Before 

No roles page available and no options to request a  role

### After

You'll see a new option on left menu to see a page with your current roles and withing the page an option to request a new role

![captura de pantalla de 2017-05-09 17 20 16](https://cloud.githubusercontent.com/assets/2192121/25876639/d21b3cb4-34db-11e7-8783-00da351fc1a5.png)



### How to test / reproduce

You'll see a more populated page of you have some roles in a couple of organizations
- Login to the app
- You'll see a roles option under My profile in left menu. Select Roles.
- You'l see a list of your current roles
- If you have pending role requests, they should be listed as well
- In each organiation you'll see an option to add a role
- You should not see roles that you alredy have in the roles options

### Related PRs
N/A

## Migrations
 NO


Thanks!
